models/marts/stg_payments.sql

```diff
@@ -0,0 +1 @@
+select * from raw.stripe.payment
```

Comment:
Do you want to move this to the staging folder? And is this just a refactoring exercise to show how you use source?